br-new

```diff
@@ -0,0 +1,230 @@
+br:
+  profile_help: |
+    A gloriosa substituição do fc! Confira !!perfil de ajuda para começar.
+  
+    Tente também !!perfil fileira de ajuda para informações mais detalhadas.
+  
+    Você tem que adicionar seu fc antes que seu perfil funcione!
+  
+    Por favor, marque "!!perfil ajuda fc" e adicione seu código de amigo antes de dizer que spyke não está funcionando! Obrigado!
+
+  level_help: |
+    
+    Defina seu nível em splatoon
+    Exemplo:
+    !!perfil nivel 50
+    Se você é um jogador de estrela, adicione um S no final após o seu nível
+    !!perfil nivel 50 S
+  level_set: seu nível foi ajustado para
```

Comment:
As an example, you've combined keys from different files - it'd be beneficial not to do that since it messes with Spyke